miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppHttpWebView.kt

```diff
@@ -0,0 +1,45 @@
+package com.rakuten.tech.mobile.miniapp.display
+
+import android.content.Context
+import com.rakuten.tech.mobile.miniapp.MiniAppInfo
+import com.rakuten.tech.mobile.miniapp.MiniAppScheme
+import com.rakuten.tech.mobile.miniapp.js.MiniAppMessageBridge
+import com.rakuten.tech.mobile.miniapp.navigator.MiniAppNavigator
+import com.rakuten.tech.mobile.miniapp.permission.MiniAppCustomPermissionCache
+import java.util.*
+
+internal class MiniAppHttpWebView(
+    context: Context,
+    miniAppTitle: String,
+    val appUrl: String,
+    miniAppMessageBridge: MiniAppMessageBridge,
+    miniAppNavigator: MiniAppNavigator?,
+    hostAppUserAgentInfo: String,
+    miniAppWebChromeClient: MiniAppWebChromeClient = MiniAppWebChromeClient(context, miniAppTitle),
+    miniAppCustomPermissionCache: MiniAppCustomPermissionCache
+) : MiniAppWebView(
+    context,
+    "",
+    MiniAppInfo.empty(),
+    miniAppMessageBridge,
+    miniAppNavigator,
+    hostAppUserAgentInfo,
+    miniAppWebChromeClient,
+    miniAppCustomPermissionCache
+) {
+    init {
+        miniAppScheme = MiniAppScheme.schemeWithCustomUrl(appUrl)
+        miniAppId = UUID.fromString(appUrl).toString() // some id is needed to handle custom permissions
```

Comment:
oh I'm sorry. `UUID.fromString(appUrl).toString()` isn't working as I expected 🙇  so we can keep your old code `"custom${Random.nextInt(0, Int.MAX_VALUE)}"` or `UUID.randomUUID().toString()`.

Reply:
`UUID.randomUUID().toString()` +1

Reply:
Ok. Fixed